Assets/Utilities/GameManager.cs

```diff
@@ -7,6 +7,7 @@ public class GameManager : Singleton<GameManager>
     public string displayName;
     public string gid;
     private List<Beastie> beasties = new List<Beastie>();
+    private Beastie selectedBeastie;
     private int selectedBeastieIndex { get; set; } = -1;

```

Comment:
We can remove this since it's no longer being used.

Reply:
Removed that and GetBeastie at index.